README.md

```diff
@@ -70,7 +70,7 @@ enable it manually:
     .setOutputPath('public/build/')
     .setPublicPath('/build')
     .setManifestKeyPrefix('build/')
-
+1
```

Comment:
I believe this was an accident? :)

Reply:
Oh no, how embarrassing - for my first and possibly smallest possible pull request I go and get it 50% wrong with my own typo!! 😲 

That'll teach me for doing it quickly directly in Github instead of cloning it locally. I was having real problems with Github doing weird stuff at the time. Anyway, yes a mistake for sure! But hey, this is a good learning experience for me to see the process of a pull request being reviewed! I shall fix!